habitat-hitl/habitat_hitl/config/hitl_defaults.yaml

```diff
@@ -34,6 +34,10 @@ habitat_hitl:
       code_available: 200  # "Ok"
       code_unavailable: 503  # "Service Unavailable"
 
+    active_features:
```

Comment:
Not a fan of this name "active_features". (Rhetorical question: what about "http_availability_server"? Isn't that a feature?)

If all your "active features" are going to be syncing or forwarding, call the group something along those lines. And I wouldn't bother with "active"; that seems meaningless.